packages/x-telemetry/README.md

```diff
@@ -0,0 +1,104 @@
+# @mui/x-telemetry
+
+Package used by some of MUI X to collects **completely anonymous** temetry data about general usage. Participation in this anonymous program is optional, and you may opt-out if you'd not like to share any information.
+
+## How to opt-in
+
+Currently, **it's disabled by default,** and you could opt-in to it in 3 ways:
+
+1. By using CLI util to enable/disable it for the current machine.
+
+```shell
+npx mui-x-telemetry enable
+```
+
+2. By setting it directly to package settings on the application start (e.g. in main file).
+
+```js
+import { muiXTelemetrySettings } from '@mui/x-telemetry';
+// or
+import { muiXTelemetrySettings } from '@mui/x-license';
+
+muiXTelemetrySettings.enableTelemetry(); // to enable telemetry collection and sending
+// or
+muiXTelemetrySettings.disableTelemetry(); // to disable telemetry collection and sending
+```
+
+3. By setting the environment variable.
+
+```dotenv
+MUI_X_TELEMETRY_ENABLED=true
+# or
+MUI_X_TELEMETRY_DISABLED=true
```

Comment:
Why not just one?

I could use only having
```suggestion
MUI_X_TELEMETRY_DISABLED=true
```

and using the value used as a forced value.

To check what the others are doing.

Reply:
Other seems to use the disabled

- `NEXT_TELEMETRY_DISABLED=1`
- `export HOMEBREW_NO_ANALYTICS=1`
- `"telemetry.telemetryLevel": "off"` (VSCode)
- `GATSBY_TELEMETRY_DISABLED to 1`